libraries/script-engine/src/FileScriptingInterface.h

```diff
@@ -35,37 +37,70 @@ class FileScriptingInterface : public QObject {
+     * @param {string} path - The path of the zip file in the local file system. May have a leading <code>"file:///"</code>. 
+     *     Need not have a <code>".zip"</code> extension if it is in a temporary directory (as created by 
+     *     {@link File.getTempDir|getTempDir}).
+     * @param {string} url - <em>Not used.</em>
+     * @param {boolean} autoAdd - <em>Not used by user scripts.</em> The value is simply passed through to the 
+     *     {@link File.unzipResult|unzipResult} signal.
+     * @param {boolean} isZip - Set to <code>true</code> if <code>path</code> has a <code>".zip"</code> extension, 
+     *     <code>false</code> if it doesn't (but should still be treated as a zip file).
+     * @param {boolean} isBlocks - <em>Not used by user scripts.</em> The value is simply passed through to the 
+     *     {@link File.unzipResult|unzipResult} signal.
+     * @example <caption>Select and unzip a file.</caption>
+     * File.unzipResult.connect(function (zipFile, unzipFiles, autoAdd, isZip, isBlocks) {
+     *     print("File.unzipResult()");
+     *     print("- zipFile: " + zipFile);
+     *     print("- unzipFiles(" + unzipFiles.length + "): " + unzipFiles);
+     *     print("- autoAdd: " + autoAdd);
+     *     print("- isZip: " + isZip);
+     *     print("- isBlocks: " + isBlocks);
+     * });
+     * 
+     * var zipFile = Window.browse("Select a Zip File", "", "*.zip");
+     * if (zipFile) {
+     *     File.runUnzip(zipFile, "", false, true, false);
+     * } else {
+     *     print("Zip file not selected.");
+     * }
      */
     void runUnzip(QString path, QUrl url, bool autoAdd, bool isZip, bool isBlocks);
 
     /**jsdoc
+     * Creates a new, unique directory for temporary use.
      * @function File.getTempDir
-     * @returns {string}
+     * @returns {string} The path of the newly created temporary directory.
+     * @example <caption>Create a temporary directory.</caption>
+     * print("New temporary directory: " + File.getTempDir());
      */
     QString getTempDir();
 
 signals:
 
     /**jsdoc
+     * Triggered when {@link File.runUnzip|runUnzip} completes.
      * @function File.unzipResult
-     * @param {string} zipFile
-     * @param {string} unzipFile
-     * @param {boolean} autoAdd
-     * @param {boolean} isZip
-     * @param {boolean} isBlocks
+     * @param {string} zipFile - The file that was unzipped.
+     * @param {string[]} unzipFiles - The paths of the unzipped files in a newly created temporary directory. Includes entries 
+     *     for any subdirectories created. An empty array if the <code>zipFile</code> could not be unzipped.
+     * @param {boolean} autoAdd - The value that {@link File.runUnzip|runUnzip} was called with.
+     * @param {boolean} isZip - <code>true</code> if {@link File.runUnzip|runUnzip} was called with <code>isZip == true</code> 
+     *     or <code>path</code> containing <code>"vr.google.com/downloads"</code>, unless there is no FBX or OBJ file in the 
```

Comment:
I'm confused here because <path> is described above as a location on the local file system. So 
```suggestion
     <code>path</code> containing <code>"vr.google.com/downloads"</code>
```
doesn't make sense to me. 
What am I missing? 

(BTW, that's the only comment I have on this PR -- good work!)

Reply:
Well spotted! I've amended the text and filed the following Jira: https://highfidelity.atlassian.net/browse/BUGZ-1466